CONTRIBUTING.md

```diff
@@ -565,6 +565,43 @@ Kubernetes-related testing are all end-to-end, living under `topgun/k8s`. They
 require access to a real Kubernetes cluster with access granted through a
 properly configured `~/.kube/config` file.
 
+[`kind`] is a great choice when it comes to running a local Kubernetes cluster -
+all you need is `docker`, and the `kind` CLI. If you wish to run the tests with
+a high degree of concurrency, it's advised to have multiple kubernetes nodes.
```

Comment:
Does this also require running `ginkgo -nodes=4`?

Reply:
nah, `topgun` does nothing smart here about that. do you think it'd be good to point out that `ginkgo -node=N` is a good thing to run if you wish to increase the concurrency?

Reply:
We mention `-node` when running testflight, for the inverse case though of reducing concurrency 😅 . I think it's worth mentioning.

Reply:
https://github.com/concourse/concourse/pull/5518/files#r420310704 :thinking: